R/parseTimeseries.R

```diff
@@ -29,7 +33,12 @@ parseTimeseries <- function(file, delim, with.units = FALSE){
                   as.is=TRUE, check.names = FALSE, stringsAsFactors = FALSE)
     #parse the date into POSIXct format
     blockData$TIMESTEP = as.POSIXct(blockData$TIMESTEP, "%Y-%m-%dT%H:%M:%S", tz="UTC")
-    statNames <- unique(names(blockData)[-1])
+    if(grepl('threshold', names(blockData)[2])){
```

Comment:
would rather do something like `!names(blockData) %in% c("threshold","TIMESTEP")` rather than assume order